talosintelligence_connector.py

```diff
@@ -59,17 +66,15 @@ def _process_html_response(self, response, action_result):
         try:
             soup = BeautifulSoup(response.text, "html.parser")
             error_text = soup.text
-            split_lines = error_text.split("\n")
+            split_lines = error_text.split('\n')
             split_lines = [x.strip() for x in split_lines if x.strip()]
-            error_text = "\n".join(split_lines)
+            error_text = '\n'.join(split_lines)
         except:
             error_text = "Cannot parse error details"
 
-        message = "Status Code: {0}. Data from server:\n{1}\n".format(
-            status_code, error_text
-        )
+        message = "Status Code: {0}. Data from server:\n{1}\n".format(status_code, error_text)
 
-        message = message.replace("{", "{{").replace("}", "}}")
+        message = message.replace(u'{', '{{').replace(u'}', '}}')
```

Comment:
It treats curly brackets rather than placeholders since error messages could have curly brackets. We do this across a lot of other connectors when processing responses too. Since it's a one liner I don't think we really need a utility function for it.